functions/src/application/members/model/MemberDto.ts

```diff
@@ -0,0 +1,12 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class MemberDto {
+  @ApiProperty()
+  firstName?: string;
+  @ApiProperty()
+  lastName?: string;
+  @ApiProperty({
+    nullable: true,
+  })
+  picture?: string;
```

Comment:
Does it exist functionally to have no picture? Why do we specify nullable for picture but not for firstName or lastName?

Reply:
in case members play but haven't set a picture yet